test/suites/ACID.m

```diff
@@ -2434,6 +2435,26 @@
     set(ax4, 'Color', 'none');
 end
 % =========================================================================
+function [stat] = histogramPlot()
+
+  if ~exist('histogram')
```

Comment:
@egeerardyn Not sure I follow. 

`isHG2 = strcmpi(m2t.env,'MATLAB') && ~isVersionBelow(m2t.env, m2t.envVersion, [8,4]);`

And [R2014b release notes on histogram](http://uk.mathworks.com/help/matlab/release-notes.html?#buf5bgh)